### ISSUE TITLE
baritone will attempt to mine unbreakable blocks (such as bedrock or portal blocks) when building

### ISSUE THREAD

**Issue by HexCodeFFF** [cabaletta/baritone#1513]
## Some information
Operating system: windows 10 64bit
Java version: Version 8 Update 241 (build 1.8.0_241-b07)
Minecraft version: 1.15.2
Baritone version: 1.5.1
Forge mods (if used): Impact is installed (not via forge)

## How to reproduce
- go into survival
- select an area containing bedrock
- use a selection command to break the area
- baritone will try to mine the bedrock
it should either fail or ignore them (ignoring would be much nicer)

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x] I have included logs, exceptions and / or steps to reproduce the issue.
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by JacksonChen666** [comment]
try adding unbreakable blocks to `blocksToAvoidBreaking` in the settings like `minecraft:bedrock`

**Comment by dominikaaaa** [comment]
That's true but maybe it should have those `blocksToAvoidBreaking` by default, as you normally can't break bedrock or barrier blocks.

**Comment by rand0m0User** [comment]
you can also tell it to mine air, witch breaks the game if you do it above Hight limit

**Comment by Tytova** [comment]
blocksToAvoidBreaking doesnt work

**Comment by ZacSharp** [comment]
Right, because it's a pathfinder setting and `BuilderProcess` will try to break anything you told it to break.
`buildIgnoreBlocks` might help, if the schematic contains air where the bedrock is.
